### PR TITLE
Add python packaging

### PR DESCRIPTION
**Pull request by alessandropalla**
Also added pyroma to check for python package quality control

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+setup(
+    name="WADAS",
+    version="0.1.0",
+    packages=find_packages(),
+    entry_points={
+        "console_scripts": ["wadas=src.main:main"],
+    },
+    author="Stefano Dellosa, Alessandro Palla",  # TODO: Add your names
+    author_email="stefano.dellosa@gmail.com, alespalla.ap@gmail.com",  # TODO: Add your emails
```
Comment (alessandropalla) [inline setup.py:17]: Please add your names/emails here

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -40,3 +40,7 @@ repos:
         name: isort (python)
         args: ["--profile", "black", "--filter-files"]
         exclude: ^src/ui/.*|demo/.*
+-   repo: https://github.com/regebro/pyroma
```
Comment (stefanodellosa-personal) [inline .pre-commit-config.yaml:43]: pyroma?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+)
+
+
+setup(
+    name="wadas",
+    version="0.1.0",
+    packages=find_packages(),
+    package_data={"": ["*.ui", "*.png", "*.jpg"]},
+    include_package_data=True,
+    author="Stefano Dellosa, Alessandro Palla",  # TODO: Add your names
```
Comment (stefanodellosa-personal) [inline setup.py:15]: author="Stefano Dell'Osa, Alessandro Palla, Antonio F., Cesare Di Mauro",

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    author_email="stefano.dellosa@gmail.com, alespalla.ap@gmail.com",  # TODO: Add your emails
```
Comment (stefanodellosa-personal) [inline setup.py:16]: author_email="stefano.dellosa@gmail.com, alespalla.ap@gmail.com, hexfati@gmail.com, cesare.di.mauro@gmail.com",

**Review by stefanodellosa-personal — APPROVED** [review]

**Pull request merged**